woocommerce-gateway-checkout-com/includes/api/class-wc-checkoutcom-utility.php

```diff
@@ -44,6 +44,9 @@ public static function valueToDecimal($amount, $currencySymbol)
             'VND',
         );
 
+        // check for decimal seperator
+        $amount = str_replace(",",".", $value);
```

Comment:
@ved-beejadhur-cko please move this logic where the issue is happening, as this can create other unexpected issues.